tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "strict": true,
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "types": ["./node_modules/user-agent-data-types"]
```

Comment:
Do we need this? I think the triple-slash directive is enough.
I just tried removing it and didn't get errors after running `yarn type-check`.

Reply:
removed it